integration/helper/namespace.go

```diff
@@ -19,6 +20,11 @@ func TempNamespace(t *testing.T, client client.Client) *corev1.Namespace {
 			GenerateName: "acorn-test-",
 			Labels: map[string]string{
 				"test.acorn.io/namespace": "true",
+				labels.AcornProject:       "true",
+			},
+			Annotations: map[string]string{
+				labels.AcornProjectDefaultRegion:    "local",
+				labels.AcornProjectSupportedRegions: "local",
```

Comment:
What if we want to test a Project with multiple supported regions - is that possible to do here or are we restricted to local?

Reply:
The local region is the only supported region in OSS.